### PR TITLE
Add gp_lttb documentation for Toolkit 1.11.0

### PR DESCRIPTION
**Pull request by WireBaron**
# Description

We added a new version of LTTB that preserves gap in the original data in toolkit 1.11.0. This is the documentation for that function.

# Links

Fixes #[insert issue link, if any]

# Writing help

For information about style and word usage, see the [style guide](https://docs.timescale.com/timescaledb/latest/contribute-to-docs/)

# Review checklists
Reviewers: use this section to ensure you have checked everything before approving this PR:

## Subject matter expert (SME) review checklist

- [ ] Is the content technically accurate?
- [ ] Is the content complete?
- [ ] Is the content presented in a logical order?
- [ ] Does the content use appropriate names for features and products?
- [ ] Does the content provide relevant links to further information?

## Documentation team review checklist

- [ ] Is the content free from typos?
- [ ] Does the content use plain English?
- [ ] Does the content contain clear sections for concepts, tasks, and references?
- [ ] Have any images been uploaded to the correct location, and are resolvable?
- [ ] If the page index was updated, are redirects required
      and have they been implemented?
- [ ] Have you checked the built version of this content?

### CODE REVIEW

File: api/gp_lttb.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
+---
+api_name: gp_lttb()
+excerpt: An implementation of our lttb algorithm that will preserve gaps in the original data
```
Comment (charislam) [inline api/gp_lttb.md:3]:
```suggestion
excerpt: Downsample data using the Largest Triangle Three Buckets algorithm while preserving gaps
```

File: api/gp_lttb.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+than the target resolution. This occurs if there are more ranges than half the
+target resolution, because each range returns at least two points.
+
+## Required arguments
+
+|Name|Type|Description|
+|-|-|-|
+|`time`|`TIMESTAMPTZ`|Time (x) value for the data point|
+|`value`|`DOUBLE PRECISION`|Data (y) value for the data point|
+|`gapsize`|`INTERVAL`|(Optional) Minimum gap size to divide input on|
```
Comment (charislam) [inline api/gp_lttb.md:44]: Could you move this to an "Optional arguments" table instead? It's a good question whether we actually need two separate tables but let's keep it consistent across all the API references for now

File: api/gp_lttb.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+ 2020-01-03 23:00:00+00 | 5.652630961100255
+ 2020-01-05 01:00:00+00 | 5.652630961100259
+ 2020-01-05 13:00:00+00 | 9.957224306869051
+ 2020-01-06 12:00:00+00 |                 0
+ 2020-01-07 10:00:00+00 |  9.82962913144534
+ 2020-01-08 00:00:00+00 | 5.000000000000004
+```
+
+[lttb]: /api/:currentVersion:/hyperfunctions/downsample/lttb/
+[hyperfunctions-timevectors]: FIXME
```
Comment (charislam) [inline api/gp_lttb.md:95]:
```suggestion
[hyperfunctions-timevectors]: /timescaledb/:currentVersion:/how-to-guides/hyperfunctions/function-pipelines/#timevectors
```

**Review by charislam — APPROVED** [review]

File: api/gp_lttb.md (hunk 2 above)
Comment (WireBaron) [reply api/gp_lttb.md]: Updated

**Pull request merged**